code/game/objects/items/storage/firstaid.dm

```diff
@@ -643,6 +643,19 @@ obj/item/storage/pill_bottle/packet
 	skilllock = SKILL_MEDICAL_DEFAULT
 	storage_flags = STORAGE_FLAGS_BOX
 	display_maptext = FALSE
+	var/icon_original
+
+/obj/item/storage/pill_bottle/packet/Initialize()
+	. = ..()
+	icon_original = icon_state
```

Comment:
This looks like it could be done with initial() rather than a var. Let me know if I am incorrect.